packages/devtools_app/lib/src/screens/memory/memory_protocol.dart

```diff
@@ -23,51 +23,51 @@ import 'memory_timeline.dart';
 class MemoryTracker {
   MemoryTracker(this.serviceManager, this.memoryController);
 
-  ServiceConnectionManager serviceManager;
+  ServiceConnectionManager? serviceManager;
 
   final MemoryController memoryController;
 
-  VmServiceWrapper get service => serviceManager?.service;
+  VmServiceWrapper? get service => serviceManager?.service;
 
-  Timer _pollingTimer;
+  Timer? _pollingTimer;
 
-  final Map<String, MemoryUsage> isolateHeaps = <String, MemoryUsage>{};
+  final Map<String, MemoryUsage> isolateHeaps = {};
```

Comment:
nit: other way around - remove type on LHS and add it back on RHS

Reply:
done